packages/ckeditor5-ui/docs/framework/guides/deep-dive/focus-tracking.md

```diff
@@ -73,13 +73,13 @@ To spice things up even more, you should also know `isFocused` will change when
 Say, for instance, your application wants to focus an editable area of CKEditor 5 when a certain action is executed (e.g. a button is clicked). To do that, use the {@link module:engine/view/view~View#focus `focus()`} method of the editing view:
 
 ```js
-editor.editing.view.focus();
+editor.focus();
 ```
 
 This snippet focuses an editable that has the selection. If the editor has not been focused yet, this will focus the very first editable. If an editor has multiple editing roots and the user was editing content, focus will be brought back where the user left off.
```

Comment:
Not only method name/path should be changed, but sometimes change like this will require you to adjust the text of the documentation.

For instance, since the changed listing shows just a method call, rather than "more complex" method accessing, how about changing the beginning of this line to:

> This method focuses

?

Reply:
Done.